var/spack/repos/builtin/packages/raxml-ng/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RaxmlNg(CMakePackage):
+    """RAxML-NG is a phylogenetic tree inference tool which uses
+    maximum-likelihood (ML) optimality criterion.
+    Its search heuristic is based on iteratively performing a series
```

Comment:
Adding an empty line:
```suggestion
    maximum-likelihood (ML) optimality criterion.

    Its search heuristic is based on iteratively performing a series
```
will make Spack use only the first paragraph in contexts where a short description is needed, and the entire docstring where instead a long description is required.